workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/Options.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using CommandLine;
+
+namespace Improbable.Gdk.DeploymentLauncher
+{
+    public class Options
+    {
+        [Verb("create", HelpText = "Create a deployment.")]
+        public class Create
+        {
+            [Option("project_name", Required = true, HelpText = "The SpatialOS project name")]
+            public string ProjectName { get; set; }
+
+            [Option("assembly_name", Required = true, HelpText = "The assembly to use in the deployment.")]
+            public string AssemblyName { get; set; }
+
+            [Option("deployment_name", Required = true, HelpText = "The name of the deployment to create.")]
+            public string DeploymentName { get; set; }
+
+            [Option("launch_json_path", Required = true, HelpText = "The path to the launch json file.")]
+            public string LaunchJsonPath { get; set; }
+
+            [Option("snapshot_path", Required = false, HelpText = "The path to the snapshot.")]
+            public string SnapshotPath { get; set; }
+
+            [Option("region", Required = true, HelpText = "The region to launch the deployment in.")]
+            public string Region { get; set; }
```

Comment:
Could this be better implemented as an enum?

Reply:
Yes - will allow us to do validation on the option parsing. Not sure what the `CommandLine` library does with enums